features/schema_metadata.feature

```diff
@@ -0,0 +1,296 @@
+      Garbage Collection Grace Seconds: 3600
+      Read Repair Chance: 0.75
+      """
+
+    @cassandra-version-less-2.1
+    Scenario: Additional table metadata will only exist for Cassandra versions
+              1.2 and v2.0
+    Given the following example:
+      """php
+      <?php
+      $cluster = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session = $cluster->connect("simplex");
+      
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Populate I/O Cache on Flush: " . ($table->populateIOCacheOnFlush() ? "True" : "False") . "\n";
+      echo "Replicate on Write: " . ($table->replicateOnWrite() ? "True" : "False") . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Populate I/O Cache on Flush: True
+      Replicate on Write: False
+      """
+
+    @cassandra-version-only-2.0
+    Scenario: Additional table metadata will only exist for Cassandra versions
+              2.0
+    Given the following example:
+      """php
+      <?php
+      $cluster = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session = $cluster->connect("simplex");
+      $cql = "ALTER TABLE values WITH index_interval = '512'";
+      $session->execute(new Cassandra\SimpleStatement($cql));
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Index Interval: " . $table->indexInterval() . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Index Interval: 512
+      """
+
+    @cassandra-version-2.0
+    Scenario: Additional table metadata will only exist for Cassandra versions
+              2.0+
+    Given the following example:
+      """php
+      <?php
+      $cluster = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session = $cluster->connect("simplex");
+      $cql = "ALTER TABLE values WITH default_time_to_live = '10000' AND "
+             . "memtable_flush_period_in_ms = '100' AND "
+             . "speculative_retry = '10ms'";
+      $session->execute(new Cassandra\SimpleStatement($cql));
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Default TTL: " . $table->defaultTTL() . "\n";
+      echo "Memtable Flush Period: " . $table->memtableFlushPeriodMs() . "\n";
+      echo "Speculative Retry: " . intval($table->speculativeRetry()) . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Default TTL: 10000
+      Memtable Flush Period: 100
+      Speculative Retry: 10
+      """
+
+    @cassandra-version-2.1
+    Scenario: Additional table metadata will only exist for Cassandra versions
```

Comment:
Scenario name suggestion - Getting table metadata for max and min index intervals